src/lib/ops/organizationOps.js

```diff
@@ -172,6 +200,11 @@ var organizationOps = {
 
     const tasks = [
       (job, next) => {
+        const { id, name, description, user } = params
+
+        Joi.validate({ id, name, description, user }, validationRules.create, next)
```

Comment:
In the code below there is an `if (err) return next(Boom.badRequest(err))` any reason no to put it also here?